arm/avhclient/aws_backend.py

```diff
@@ -688,10 +687,10 @@ def create_or_start_instance(self) -> AvhBackendState:
         if self.instance_id:
             state = self.get_instance_state()
             if state == "running":
-                logging.info(f"aws:EC2 Instance {self.instance_id} already running!")
+                logging.debug(f"aws:EC2 Instance {self.instance_id} already running!")
                 return AvhBackendState.RUNNING
             if state == "stopped":
-                logging.info(f"aws:EC2 Instance {self.instance_id} provided!")
+                logging.debug(f"aws:EC2 Instance {self.instance_id} provided!")
```

Comment:
f-strings are not recommended with logging, use individual params instead (results in lazy formatting)
```suggestion
                logging.debug("aws:EC2 Instance %s provided!", self.instance_id)
```